docs/datastore/ref-getting-started.md

```diff
@@ -0,0 +1,141 @@
+---
+id: getting-started
+title: Getting Started
+sidebar_label: Getting Started
+---
+
+## Installing Offix DataStore
+
+Using [npm](https://www.npmjs.com/package/offix-datastore):
+
+```shell
+npm install offix-datastore
+```
+
+Or [yarn](https://yarnpkg.com/en/package/offix-datastore):
+
+```shell
+yarn add offix-datastore
+```
+
+## Using DataStore
+
+### Sample GraphQL Schema
+
+Let's use the following sample schema for our app
+
+```
+type Task {
+    id: ID!
+    title: String
+    description: String
+    numberOfDaysLeft: Number
+}
+
+```
+
+### Configuring Datastore
+
+We have a [cli tool](cli.md) that generates DataStore config and Model JSON schema given a GraphQL schema.
+`DataStore` accepts a config object and an optional `CustomEngines`, see [api reference](datastore-api).
+
+### Datastore Models
+
+To be able to store user tasks in DataStore, you need to create its DataStore model.
+The DataStore model provides the API to perform CRUD operations on `Task` in the DataStore.
+
+Let's create the Task DataStore model.
```

Comment:
This is oudated. We should create guide how to generate this instead of creating this by hand :)

Reply:
I thought the manual guide would still be of use even with the cli guide. We can take out this part.

Reply:
CLI is esentiall. Telling people to do tons of boilerplate makes no sense.